containers/README.md

```diff
@@ -0,0 +1,46 @@
+# Container files
+
+## Running the production containers
+
+Container images are automatically published to
+[this repository's GitHub Packages registry](https://github.com/unioslo/tsd-file-api/pkgs/container/tsd-file-api).
+
+## Build notes
+
+All these containers expect the repository root as their build context, i.e.
+the positional parameter you pass when building with `podman`/`docker`.
+
+An example (with `$PWD` being the parent directory to this one):
+
+```console
+podman build -f containers/Dockerfile.alpine .
+```
+
+Another example (passing the repository URL as context):
+
+```console
+podman build -f containers/Dockerfile.alpine https://github.com/unioslo/tsd-file-api.git
+```
+
+## Container overview
+
+### `Dockerfile`
+
+Production ready service container using that is using
```

Comment:
"using that is using" -> "using"

Reply:
There are worse issues IMO, like the explanation of the example invocation in the main README not being consistent with it: https://github.com/unioslo/tsd-file-api#running-the-service

I just can't seem to write anything consistent today... Will fix later.